apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -1881,11 +2044,15 @@ static int fs_sample_launcher(int argc, char **args)
 	libc_stdio_fgetc_tc();
 	libc_stdio_fwrite_tc();
 	libc_stdio_fread_tc();
+	libc_stdio_freopen_tc();
 	libc_stdio_ferror_tc();
 	libc_stdio_clearerr_tc();
 	libc_stdio_gets_tc();
 	libc_stdio_gets_s_tc();
 	libc_stdio_fileno_tc();
+	libc_stdio_remove();
+	libc_stdio_setbuf();
```

Comment:
setbuf and setvbuf needs CONFIG_STDIO_BUFFER_SIZE > 0

Reply:
done